README.md

```diff
@@ -30,6 +30,10 @@ To configure the annotations plugin, you can simply add new config options to yo
             // See http://www.chartjs.org/docs/#advanced-usage-creating-plugins
             drawTime: 'afterDatasetsDraw', // (default)
 
+            // Defines when annotations' labels are drawn.  Defaults to
+            // drawTime.
+            labelDrawTime: 'afterDatasetsDraw', // (default)
```

Comment:
@joshkel may I ask you why to use a property at element level instead of using `drawTime` property at `label` node?